index.js

```diff
@@ -44,13 +44,16 @@ const audits = [
   require('./audits/manifest/start-url')
 ];
 
-Gatherer
-    .gather(gatherers, {url, driver})
-    .then(artifacts => Auditor.audit(artifacts, audits))
-    .then(results => {
-      console.log(results);
-    }).catch(function(err) {
-      console.log('error encountered', err);
-      console.log(err.stack);
-      throw err;
-    });
+module.exports = function(opts) {
+  var url = opts.url || defaultUrl;
+  Gatherer
+      .gather(gatherers, {url, driver})
+      .then(artifacts => Auditor.audit(artifacts, audits))
+      .then(results => {
+        console.log(results);
```

Comment:
eventually will we want to pipe this somewhere, with the default being pipe it to console? Unless we want it to always be stdout and the user is in charge of piping